src/database-fs.ts

```diff
@@ -4,7 +4,7 @@ import { path } from './deps.ts'
 import { checkKey } from './utils.ts'
 
 // Initialized in `initStorage()`.
-let dataFolder = ''
+export let dataFolder = ''
```

Comment:
Hmm, why is this being exported? 🤔 

I don't see it imported anywhere... and isn't kind of a weird thing to do, to export such a variable?

Reply:
> I don't see it imported anywhere...

It's available as `backend.dataFolder` once the whole module has been imported as `backend`. It's not meant to be imported separately.

> why is this being exported?

We might not need this in common use cases, but in some `getBackend` tests where the same backend object is "recycled" to use a new data folder, I could not easily make sure it had been correctly updated without having access to this propery.

Of course this happens because in this design, the backend object is the result of exporting its whole module, like `export * as fs from './src/database-fs.ts'`. In other words, the module acts as a singleton instance in OOP terminology, in which case exporting a module variable is like making a property of that singleton public rather than keeping it private.
So I guess it isn't that weird in the context of that design. But maybe that design itself is a bit weird.

By the way, the ES module spec ensures an exported `let` binding is read-only outside its module (although mutable from within it)